lbrynet/lbrynet_daemon/Daemon.py

```diff
@@ -1459,19 +1461,22 @@ def jsonrpc_claim_show(self, name=None, txid=None, nout=None, claim_id=None):
     @AuthJSONRPCServer.auth_required
     @defer.inlineCallbacks
     @AuthJSONRPCServer.flags(force='-f')
-    def jsonrpc_resolve(self, uri, force=False):
+    def jsonrpc_resolve(self, force=False, uri=None, uris=[]):
         """
         Resolve a LBRY URI
 
         Usage:
-            resolve <uri> [-f]
+            resolve (<uri> | --uri=<uri>) [<uris>...] [-f]
```

Comment:
yeah, [-f] should be at the beginning.

BC aside, it's (<uri>) [<uris>...] also so that at least one uri is required